.github/workflows/linux-eic-shell.yml

```diff
@@ -564,10 +572,7 @@ jobs:
         platform-release: "${{ env.platform-release }}"
         setup: /opt/detector/setup.sh
         run: |
-          echo "::group::pip install"
-          python3 -m pip install -r ${{ github.workspace }}/.github/requirements.txt
-          echo "::endgroup::"
-          python3 ${{ github.workspace }}/epic-capybara/bara.py ref/rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root
+          epic-capybara bara ref/rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root
```

Comment:
Unfortunately, bara can not yet replace bara.py, the new CLI argument set is much nicer, but the stdout output is not the same. I would prefer if we run both bara.py script and tool in parallel for a while.

Reply:
Since `capybara` is centrally installed, all runtime dependencies are available. I'll remove the `pip install` of requirements and maintain the previous direct call of `bara.py`.